OWNERS

```diff
@@ -5,6 +5,7 @@ approvers:
   - harshad16
   - mayaCostantini
   - sesheta
+  - xtuchyna
```

Comment:
```suggestion
  - xtuchyna
  - goern 
```

somehow we missed including Christoph, can you please add him too only on the approver's list.

Reply:
All good now @harshad16 :+1: